symengine/parser.cpp

```diff
@@ -370,13 +503,13 @@ class ExpressionParser
         std::stack<std::pair<int, unsigned int>> op_stack;
 
         bool last_char_was_op = false;
-        char last_char = 'x';
+        std::string last_char = "x";
         s.clear();
         s.reserve(in.length());
 
-        // Replacing ** with ^
         for (unsigned int i = 0; i < in.length(); ++i) {
             if (in[i] == '*' and i + 1 < in.length() and in[i + 1] == '*') {
+                // Replacing ** with ^
                 s += '^';
```

Comment:
@isuruf @srajangarg Should this be removed? Is there a need to parse `&`, `|` or `^`?